en/docs/samples/AmazonS3SinkSample.md

```diff
@@ -1,45 +1,50 @@
 # Publishing Aggregated Events to the Amazon AWS S3 Bucket
 
-## Purpose:
+## Purpose
 
-This application demonstrates how to publish aggregated events to Amazon AWS S3 bucket via the [siddhi-io-s3 sink
-extension](https://siddhi-io.github.io/siddhi-io-s3/). In this sample the events received to the `StockQuoteStream` stream are aggregated by the
+This application demonstrates how to publish aggregated events to Amazon AWS S3 bucket via the [siddhi-io-s3 sink](https://siddhi-io.github.io/siddhi-io-s3/) extension. In this sample the events received to the `StockQuoteStream` stream are aggregated by the
 `StockQuoteWindow` window, and then published to the Amazon S3 bucket specified via the `bucket.name` parameter.
 
 !!!info "Before you begin:"
-    - Create an AWS account and set the credentials following the instructions in the [AWS Developer Guide](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html).<br/>
-    - Save the sample Siddhi application in Streaming Integrator Tooling.<br/>
+    - Create an AWS account and set the credentials following the instructions in the [AWS Developer Guide](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html).
+    - Save the sample Siddhi application in Streaming Integrator Tooling.
 
+## Prerequisites
 
-## Executing the Sample:
+1. Download the following files.
+    - `woodstox-core-7.1.0.jar`
+    - `woodstox-osgi-3.2.1.1.jar`
+2. Navigate to {SI-Home}/bin and issue the following command to convert the `woodstox-osgi-3.2.1.1.jar` to OSGi Bundle.
+    - For Linux: `./jartobundle.sh <source-path> <destination-path>`
+    - For Windows: `./jartobundle.bat <source-path> <destination-path>`
+3. Copy the above two bundles to the `<SI-Tooling-Home>/lib` directory.
+
+## Executing the Sample
 
 To execute the sample, follow the procedure below:
 
 1. In Streaming Integrator Tooling, click Open and then click **AmazonS3SinkSample.siddhi** in the **workspace** directory. Then update it as follows:
 
     1. Enter the credential.provider class name as the value for the `credential.provider` parameter. If the class is not specified, the default credential provider chain is used.
 
-    2. For the `bucket.name` parameter, enter `AWSBUCKET` as the value.
+    2. Update the value of the `bucket.name` parameter.
```

Comment:
bucket.name is in the annotation right? Shall we refer to these properties mentioning where they are? also we need to mention what this property means. for example it refers to the bucket name in aws (eventhough they are self-explanatory)

Reply:
Updated